core/domain/email_manager.py

```diff
@@ -345,9 +345,10 @@ def send_job_failure_email(job_id):
     send_mail_to_admin(mail_subject, mail_body)
     other_recipients = (
         NOTIFICATION_EMAILS_FOR_FAILED_TASKS.value)
+    system_name_email = '%s <%s>' % ('SYSTEM', feconf.SYSTEM_EMAIL_ADDRESS)
```

Comment:
Move this system name to feconf. Also, we are assuming the email to be in this pattern so maybe you can change `system_name_email` --> `system_email`.

Reply:
Moving system name to feconf will allow software owner to configure the system name.

Reply:
I have a slightly different proposal here. Let's do:

`system_name_email = '%s <%s>' % (feconf.SYSTEM_EMAIL_NAME, SYSTEM_EMAIL_ADDRESS)

and let's have SYSTEM_EMAIL_NAME be ''. (We will update it to the right value on deployment.)

Reply:
That's great. I will do the suggested changes. Thanks!

Reply:
Done!